inference-engine/src/inference_engine/include/openvino/runtime/infer_request.hpp

```diff
@@ -55,20 +56,76 @@ class INFERENCE_ENGINE_API_CLASS(InferRequest) {
      * @brief Sets input/output data to infer
      *
      * @note Memory allocation does not happen
-     * @param name Name of input or output blob.
-     * @param data Reference to input or output blob. The type of a blob must match the network input precision and
+     * @param name Name of input or output tensor.
+     * @param tensor Reference to input or output tensor. The type of a tensor must match the network input precision
+     * and size.
+     */
+    void set_tensor(const std::string& name, const Tensor& tensor);
+
+    /**
+     * @brief Sets input data to infer
+     *
+     * @note Memory allocation does not happen
+     * @param idx Index of input tensor.
+     * @param tensor Reference to input tensor. The type of a tensor must match the network input precision and
+     * size.
+     */
+    void set_input_tensor(const std::size_t idx, const Tensor& tensor);
```

Comment:
Why `const std::size_t` in other places we are using just `size_t`.

Reply:
replaced